package.json

```diff
@@ -65,14 +64,17 @@
         "prettier": "^3.1.0",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
-        "react-router-dom": "^5.3.4",
-        "styled-components": "^5.3.3"
+        "react-router-dom": "^6.0.0",
+        "styled-components": "^6.0.0"
     },
     "peerDependencies": {
+        "@strapi/design-system": "2.0.0-rc.11",
+        "@strapi/icons": "2.0.0-rc.11",
+        "@strapi/utils": "5.2.0",
```

Comment:
Make sure these peerDependencies allow a range of versions. By strictly setting the version to `5.2.0` you restrict this plugin to begin used only with that version of Strapi.

Again, you can take the boilerplate as an example.